internal/impl/devbox.go

```diff
@@ -322,30 +322,35 @@ func (d *Devbox) PrintEnv(opts *devopt.PrintEnv) (string, error) {
 		return "", err
 	}
 
-	if opts.OnlyPathWithoutWrappers {
-		path := []string{}
-		for _, p := range strings.Split(envs["PATH"], string(filepath.ListSeparator)) {
-			if !strings.Contains(p, plugin.WrapperPath) {
-				path = append(path, p)
-			}
-		}
-
-		// reset envs to be PATH only!
-		envs = map[string]string{
-			"PATH": strings.Join(path, string(filepath.ListSeparator)),
-		}
-	}
-
 	envStr := exportify(envs)
 
-	if opts.IncludeHooks {
+	if includeHooks {
 		hooksStr := ". " + d.scriptPath(hooksFilename)
 		envStr = fmt.Sprintf("%s\n%s;\n", envStr, hooksStr)
 	}
 
 	return envStr, nil
 }
 
+// OnlyPathWithoutWrappers is a small utility to filter WrapperBin paths from PATH
+func OnlyPathWithoutWrappers() string {
+
+	path := []string{}
+	for _, p := range strings.Split(os.Getenv("PATH"), string(filepath.ListSeparator)) {
```

Comment:
os.Getenv("PATH") works because OnlyPathWithoutWrappers is used only in the context of direnv enabled or inside devbox shell. So the PATH is guaranteed to be a devbox computed PATH.
But it won't work outside that context, if you just run the command you'll get PATH env variable (assuming you don't have devbox global).
So I worry this might be used in future in a different context with the assumption that it will compute a devbox PATH and exclude wrappers bin and introduce a bug.

Reply:
1. The `devbox shellenv only-path-without-wrappers` command is Hidden, and I can add more comments to clarify its usage. It'll only be used by someone developing devbox code. So, this is low risk.

2. The internal function could be renamed to `ExportSystemPathWithoutWrappers`. WDYT?